content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -125,6 +125,10 @@ cgroup driver instead of `cgroupfs`.
 
 ### systemd cgroup driver {#systemd-cgroup-driver}
 
+{{< note >}}
+The minimum `systemd` version should be v228 and above, see [details](https://github.com/systemd/systemd/commit/17f62e9bd00f5fefd486475861b06d3ec6b7ee10)
```

Comment:
Hi @pegasas, can we make it a bit new-user friendly also, like adding a document what changes are being introduced in systemD.

Reply:
I am not really get your point.
If you are talking we should add some description about how systemd cgroup driver integrated into CRI, it would a large topic which deserves another issue for tracking.
If you are talking which specific change are introduced by systemd v228, 
I think it is unneccsary, assume if you are a user, 
what I concern is how I could use this function as soon as possible.
If indeed the customer has such kinds of demands, he/she will looks deep into source code just like the discussions on issues.

Reply:
@pegasas, thanks for the clarity, was talking about any discussion before/after this commit, if there are any?

Reply:
no, actually.
What are your suggestion?
maybe you're talking #1?

Reply:
I agree with @utkarsh-singh1 here. Not everyone can understand the code and a purpose of certain code change. How about pointing to the [systemd release notes](https://github.com/systemd/systemd/blob/v228/NEWS) ?

Here is how the change is explained there:
```
Slice units may now be created transiently via the bus APIs,
similar to the way service and scope units may already be
created transiently.
```

This is also requires some background for understanding, but it's much better than pointing to the code in my opinion.

Reply:
@pegasas  I would recommend linking to the systemd release notes or [the releases page](https://github.com/systemd/systemd/releases/tag/v228) than pointing to the actual code. Also, a small nit

```suggestion
The minimum `systemd` version should be v228 or above, see [details](https://github.com/systemd/systemd/releases/tag/v228)
```

Reply:
@divya-mohan0209 You suggested a link to the page with containerd v228 tarballs. Is that what you really wanted to point out users to?

Reply:
May be some explanations should be added to clarify why the change is important for Kubernetes, wdyt?

Reply:
Stating the required version should be sufficient IMO. What are some examples of the the explanations you'd like to see?

Reply:
I simply want to know why v228 is a minimum required version.
Here is an example: https://kubernetes.io/docs/concepts/architecture/cgroups/#using-cgroupv2
That documentation not just recommends certain software versions, but gives enough info for a reader to understand a reasoning behind it, e.g. CRI-O >= v1.20 and Containerd >= v1.4 support cgroup2.

Reply:
That's a good point. AFAIK the minimum required version is v228 per the linked issue because the kubelet fails to start with lower versions.